docs/markdown/snippets/rust-bindgen-cpp.md

```diff
@@ -0,0 +1,6 @@
+## Bindgen will now use Meson's hueristic for what is a C++ header
+
+Bindgen natively assumes that a file with the extension `.cpp` is a C++ header,
```

Comment:
Just noticed this, but oops... this should be "hpp", not "cpp". The commit message and the code comments themselves say hpp, and cpp makes no sense anyway...